java/api/openapi.yaml

```diff
@@ -558,7 +558,7 @@ paths:
         style: form
```

Comment:
Is this file auto generated from the json?

Reply:
Yeah, this is auto generated from the JSON. Good catch BTW, will do more digging why it needs to generate this file. I think just the JSON one should be enough. Added to the Java SDK ticket. https://linear.app/kanmon/issue/ENG-1420/create-java-sdk